LayoutTests/http/wpt/credential-management/setDigitalCredentialsEnabled.https.html

```diff
@@ -10,10 +10,6 @@
         testRunner.dumpAsText();
         testRunner.waitUntilDone();
 
-        if (window.navigator.credentials.requestIdentity !== undefined) {
```

Comment:
Would be nice to replace with a check that the function does NOT exist.

Reply:
ah, yeah. Will add.